Makefile

```diff
@@ -53,3 +59,15 @@ coverage: venv_check install_dev
 
 coverage-ci: COVERAGE_REPORT_FORMAT=xml
 coverage-ci: coverage
+
+build-container: 
+	${CONTAINER_ENGINE} build -t ${IMAGE} -f ${DOCKERFILE} ${CONTEXT}
+
+run-container:
+	${CONTAINER_ENGINE} run -it --rm -p ${HOST_WEBPORT}:${CONTAINER_WEBPORT} ${IMAGE} runserver 0.0.0.0:8000
```

Comment:
@mjholder  FYI:

the "0.0.0.0:8000'  bit is for rootless podman

see https://github.com/containers/podman/issues/9746#issuecomment-802702472

you might not need it with docker, I still need to figure out a fancy way to remove that (probably just another Makefile variable)